runtime/minor_gc.c

```diff
@@ -657,12 +657,12 @@ void caml_empty_minor_heap_promote(caml_domain_state* domain,
 }
 
 void caml_do_opportunistic_major_slice
-  (caml_domain_state* domain_unused, void* unused)
+  (caml_domain_state* domain, void* unused)
```

Comment:
Nit: What is the reasoning behind this renaming? `domain_unused` name indicated that the variable was unused. It still remains unused in this PR. FWIW, there are other `domain_unused` parameters in the same file. If the plan is to change all `domain_unused` to `domain` it would be useful to do it wholesale and in a separate PR.